src/InstagramClone.Api/Database/InstagramCloneDbContext.cs

```diff
@@ -10,10 +11,12 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
             optionsBuilder.UseSqlServer("DefaultConnectionString");
         }
```

Comment:
... and here too.

When it says "This means that the configuration passed to AddDbContext will never be used"

it's saying that it knows that you have probably configured your DbContext another way than it expects.

So one way that I think will resolve this will be to refactor the dbcontext configuration in the Startup class and only call `AddDbContext<XXXX>()` (without any options)

Reply:
Yes, I just did this and it works. Thanks a lot.